### PR TITLE
documentation #530: added Swagger documentation to Group endpoints

### LINKED ISSUE #530: Add Swagger documentation to Group endpoints (1)
Separate the section with group endopints and describe the documentation of **exactly those below**:

image

### PR DESCRIPTION
**Pull request by JessFreak**
- described swagger documentation for create method
- described swagger documentation for getAll method
- described swagger documentation for get method
- described swagger documentation for update method
- described swagger documentation for deleteGroup method

closes #530

### CODE REVIEW

File: src/v2/api/controllers/GroupController.ts [hunk 1]
```diff
@@ -24,20 +27,55 @@ export class GroupController {
     private groupMapper: GroupMapper,
   ) {}
 
+  @ApiBearerAuth()
+  @ApiOkResponse({
+    type: GroupResponse,
+  })
+  @ApiBadRequestResponse({
+    description: `
+      InvalidBodyException:  Code can not be empty\n
```
Comment (kropli) [inline src/v2/api/controllers/GroupController.ts:36]: You do not need newline character in such list of exceptions

File: src/v2/api/controllers/GroupController.ts [hunk 2]
```diff
@@ -24,20 +27,55 @@ export class GroupController {
...
     const group = await this.groupService.create(body.code);
     return this.groupMapper.getGroup(group);
   }
 
+  @ApiOkResponse({
+    type: GroupsResponse,
+  })
+  @ApiBadRequestResponse({
+    description: `
+      InvalidBodyException: Page must be a number\n
```
Comment (kropli) [inline src/v2/api/controllers/GroupController.ts:57]: Same problem as mentioned higher in code

File: src/v2/api/controllers/GroupController.ts [hunk 3]
```diff
@@ -46,6 +84,22 @@ export class GroupController {
...
   }
 
+  @ApiBearerAuth()
+  @ApiOkResponse({
+    type: GroupResponse,
+  })
+  @ApiBadRequestResponse({
+    description: `
+      InvalidEntityIdException: Group with such id is not found\n
+      InvalidBodyException: Code can not be empty\n
```
Comment (kropli) [inline src/v2/api/controllers/GroupController.ts:94]: Same problem as mentioned higher in code

File: src/v2/api/dtos/CreateGroupDTO.ts [hunk 4]
```diff
@@ -1,7 +1,9 @@
 import { IsNotEmpty, Matches } from 'class-validator';
 import { validationOptionsMsg } from '../../utils/GLOBALS';
+import { ApiProperty } from '@nestjs/swagger';
 
 export class CreateGroupDTO {
+  @ApiProperty()
   @Matches(
```
Comment (kropli) [inline src/v2/api/dtos/CreateGroupDTO.ts:7]: Match decorator do not described in swager

File: src/v2/api/dtos/UpdateGroupDTO.ts [hunk 5]
```diff
@@ -1,7 +1,9 @@
 import { IsNotEmpty, Matches } from 'class-validator';
 import { validationOptionsMsg } from '../../utils/GLOBALS';
+import { ApiProperty } from '@nestjs/swagger';
 
 export class UpdateGroupDTO {
+    @ApiProperty()
     @Matches(/І[МПКАСОВТ]-([зпв]|зп)?\d\d(мн|мп|ф)?і?/, validationOptionsMsg('Proper name is expected'))
```
Comment (kropli) [inline src/v2/api/dtos/UpdateGroupDTO.ts:7]: Match decorator do not described in swager

File: src/v2/api/controllers/GroupController.ts [hunk 6]
```diff
@@ -46,6 +84,22 @@ export class GroupController {
...
+      InvalidBodyException:  Code must be a string
```
Comment (kropli) [inline src/v2/api/controllers/GroupController.ts:95]: Incorrect exception

**Review by kropli — APPROVED** [review]

**Review by kropli — APPROVED** [review]

**Review by f-ogoog — APPROVED** [review]

**Review by just-hoshinon — APPROVED** [review]

**Pull request merged**